### PR TITLE
Add osu!mania 4K World Cup 2022 Newspost and revise Wiki entry

### PR DESCRIPTION
**Pull request by ChillierPear**
Keyboard time

### CODE REVIEW

**Review by Blooshing — COMMENTED** [review]
Otherwise it seems good to me.

**Comment by Ephemeralis** [comment]
Needs an Open Collective event made for MWC and the link changed to that before this can proceed to merge.

File: news/2022-07-07-osu-mania-4k-world-cup-2022-registrations-now-open.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+| Round of 32 | Aug 13th & Aug 14th |
+| Round of 16 | Aug 20th & Aug 21st |
+| Quarterfinals | Aug 27th & Aug 28th |
+| Semifinals | Sep 3rd & Sep 4th |
+| Finals | Sep 10th & Sep 11th |
+| Grand Finals | Sep 17th & Sep 18th |
+
+## Prizes
+
+The osu!mania 4K World Cup offers a $2,000 cash prize pool and limited edition merch!
```
Comment (peppy) [inline news/2022-07-07-osu-mania-4k-world-cup-2022-registrations-now-open.md:37]:
```suggestion
The osu!mania 4K World Cup offers a minimum $2,000 cash prize pool and limited edition merch!
```

**Comment by peppy** [comment]
Please do NOT make pull requests from your `master` branch.

**Review by peppy — APPROVED** [review]
.

**Pull request merged**